app/assets/stylesheets/govuk_publishing_components/components/_autocomplete.scss

```diff
@@ -0,0 +1,49 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-autocomplete {
+  position: relative;
+
+  > div {
+    margin-bottom: 0;
+  }
+  
+  .gem-c-autocomplete-result-list {
+    border: 1px solid $govuk-border-colour;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    overflow-y: auto;
+    -webkit-overflow-scrolling: touch;
+    background: govuk-colour("white");
+    list-style: none;
+    position: relative;
+  }
+  
+  .gem-c-autocomplete-result {
```

Comment:
Using BEM means these classes shouldn't need nesting (something we try to avoid for inheritance reasons) so I think you should be able to structure this file more like:

```CSS
.gem-c-autocomplete {
  ...
}

.gem-c-autocomplete__result {
  ...
}
```